src/script/script.js

```diff
@@ -40,7 +40,7 @@ Object.assign(pc, function () {
                 if (value instanceof pc.GraphNode) {
                     return value;
                 } else if (typeof value === 'string') {
-                    return app.root.findByGuid(value);
+                    return app._entityIndex[value];
```

Comment:
I think we should at least add a undocumented method to access this, rather than directly access the private variable